### PR TITLE
Remove id and nullable options from definitions

### PR DESCRIPTION
**Pull request by benediktdertinger**
Remove id and nullable options from `definitions.User` to avoid TS schema error.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,10 @@ fastify.register(require('fastify-swagger'), {
     ],
     definitions: {
       User: {
-        $id: 'User',
         type: 'object',
         required: ['id', 'email'],
         properties: {
           id: { type: 'string', format: 'uuid' },
-          firstName: { type: 'string', nullable: true },
-          lastName: { type: 'string', nullable: true },
```
Comment (mcollina) [inline README.md:51]: Can you just remove `nullable: true` and not the fields?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (benediktdertinger) [reply README.md:51]: Yep!

**Review by fox1t — APPROVED** [review]
LGTM

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**